pkgs/development/libraries/libjpeg-turbo/default.nix

```diff
@@ -42,9 +42,13 @@ stdenv.mkDerivation (finalAttrs: {
   };
 
   # This is needed by freeimage
-  patches = [ ./0001-Compile-transupp.c-as-part-of-the-library.patch ]
-    ++ lib.optional stdenv.hostPlatform.isMinGW
-    ./mingw-boolean.patch;
+  patches =
+    lib.optionals (!stdenv.hostPlatform.isMinGW) [
+      ./0001-Compile-transupp.c-as-part-of-the-library.patch
+    ] ++ lib.optionals stdenv.hostPlatform.isMinGW [
+      ./mingw-boolean.patch
+      ./transup-as-part-without-attribute-weak.patch
```

Comment:
Can this be done via a `fetchPatch` from https://github.com/libjpeg-turbo/libjpeg-turbo/commit/537b9c336e8ba7f128b4171388b5d1b5b39f53ea? It would also be helpful to include the link to https://github.com/libjpeg-turbo/libjpeg-turbo/pull/487.